### PR TITLE
initial scrapper

### PR DESCRIPTION
**Pull request by MandyMeindersma**
@GarryBullock 

DO NOT MERGE

### CODE REVIEW

File: bfex/components/scrapper/scrapper.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# this is where we would do json parsing to get the names?!?!?!
+
+# concatenate names into url? here ?!?!!?
+
+url = "https://www.ualberta.ca/science/about-us/contact-us/faculty-directory/william-allison"
+response = http.request("GET", url)
+soup = BeautifulSoup(response.data, "html.parser")
+links = soup.find_all("a")
+for link in links:
+    if 'ORCID ID' in link.text:
```
Comment (GarryBullock) [inline bfex/components/scrapper/scrapper.py]: I would not do it this way. I've seen examples where professors had tagged the link differently. ie 'OrcID' 'OrcID ID'

I would look at the URL of the link, as it should point to the Orc website, and you can extract that id from there.

**Comment by GarryBullock** [comment]
Also, can you look at using requests instead of urllib3? It's the one we had said we would use (its in the high level docs), is more convenient, and easier to read.

[This kind of summarizes why I think we should use it](https://stackoverflow.com/questions/2018026/what-are-the-differences-between-the-urllib-urllib2-and-requests-module)

**Comment by MandyMeindersma** [comment]
we have urllib3 in our conda file @GarryBullock ?

**Comment by MandyMeindersma** [comment]
but yeah ill change that, you are right

**Comment by GarryBullock** [comment]
Yeah, urllib3 is used by another module, I believe flask.

Also, you spelled scraper wrong, lol.

**Comment by MandyMeindersma** [comment]
I DID NOT SPELL IT WRONG, ITS IN THE GLOSSARY AS SCRAPPER

**Comment by GarryBullock** [comment]
Lol. I didnt add that

**Pull request closed without merging**